lib/remove_duplicate_emails.rb

```diff
@@ -0,0 +1,36 @@
+class RemoveDuplicateEmails
+  def run(do_it: false)
+    # find all duplicate contact infos, older than X
+    all_dup_contact_infos = ContactInfo.where("created_at < ?", Date.parse("1/10/2020")).select(:type, :value).group(:type, :value).having('count(*) > 1')
```

Comment:
Maybe put the date as a parameter with default '1/10/2020' so we can run it again later without having to make another release.